src/test/java/com/rackspacecloud/metrics/ingestionservice/InfluxDBHelperTests.java

```diff
@@ -151,6 +152,9 @@ private void successfulIngestionTest(
         doNothing().when(influxDBMock).write(anyString(), anyString(), any(), any(), anyString());
 
         influxDBHelper.ingestToInfluxDb(payloadToIngestInInfluxDB, tenantId, measurement, rollupLevel);
+
+        verify(influxDBMock, Mockito.times(1))
```

Comment:
Please have it verify the actual arguments instead of using `any()`. `any()` should only be used for `when` declarations.

Check out https://github.com/racker/salus-telemetry-monitor-management/blob/master/src/test/java/com/rackspace/salus/monitor_management/services/MonitorManagementTest.java if you want to see some various uses of verify and asserts.

Reply:
Done